deps/spidershim/test/v8engine.h

```diff
@@ -95,6 +95,14 @@ class V8Engine {
     return Local<Value>();
   }
 
+  Local<Value> CompileRun(Local<Context> context, Local<String>* script) {
```

Comment:
Passing `Local` by pointer doesn't make sense.  Did you mean to pass it by value?